sdk/resourcemanager/connectedvmware/armconnectedvmware/README.md

```diff
@@ -57,7 +57,7 @@ clientFactory, err := armconnectedvmware.NewClientFactory(<subscription ID>, cre
 A client groups a set of related APIs, providing access to its functionality.  Create one or more clients to access the APIs you require using client factory.
 
 ```go
-client := clientFactory.NewVirtualMachinesClient()
+client := clientFactory.NewVCentersClient()
```

Comment:
Is this auto generated or need manual change?

Reply:
manual change